### DISCUSSION TITLE
Import org-mode knowledge base

### DISCUSSION THREAD

**Discussion by chuwy** [nvim-neorg/neorg#572] [General]
org-mode is almost a standard de-facto for personal KB and neorg's main competitor. I think it makes a lot of sense to alleviate migration from it. I currently have around thousand documents, most of them are org-roam-based and have no idea how to move that KB to neorg.

I was thinking to use some intermediate format, but apparently the same problem exists for Markdown (https://github.com/nvim-neorg/neorg/discussions/428).

The neorg file format is so awesome, takes the best from both markdown and org-mode, but the neorg NeoVim plugin it seems remains the only software that can operate it. It would really be great to see a more wide-spread adoption in tools like Pandoc.

**Comment by XelorR** [comment]
+1

I use emacs org-mode and org-roam many years. Currently I also have more then thousand cross-linked notes with both file and id links and I use most of existing org-mode functions...

Org is a standard, easily handled via external tools like LogSeq, Orgmode.nvim or VSCode plugins. It is also relatively easy to convert between Org-mode and different Markdown dialects...

To popularize and widespread ABSOLUTELY AWESOME NEORG FORMAT it is necessary to have tools and workflow of transconverting between org-mode, org-roam, popular markdown dialects and neorg.

I can test any solutions on my huge database if you find it helpful.

**Reply by pysan3** [reply]
https://github.com/nvim-neorg/neorg/discussions/572#discussioncomment-7810373

**Comment by pysan3** [comment]
I've created a cli tool to import other format into norg via pandoc. 

https://github.com/pysan3/minorg

I haven't tested for org-mode and I don't have any experience with it so it would be so nice if you could test it against your notes, and if you find anything lacking, please let me know. 

I'm also on the neorg discord server (pysan3) so ping me if you want a conversation.
